experimental/PropertyDDS/packages/property-proxy/index.js

```diff
@@ -5,7 +5,7 @@
 'use strict';
 
 if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./dist/lib/@fluid-experimental/property-proxy.min.js');
+  module.exports = require('./dist/index.js');
```

Comment:
So now we provide single build for both environments ?

Reply:
Yes, I switched from webpack to tsc. This means that the files are not minified any longer.